src/overlays/actors/ovl_En_Bird/z_en_bird.h

```diff
@@ -12,19 +12,19 @@ typedef struct EnBird {
     /* 0x0000 */ Actor actor;
     /* 0x014C */ SkelAnime skelAnime;
     /* 0x0190 */ EnBirdActionFunc actionFunc;
-    /* 0x0194 */ u32 unk_194;
-    /* 0x0198 */ s32 unk_198;
-    /* 0x019C */ s16 unk_19C;
+    /* 0x0194 */ u32 unk_194; // zero'd but otherwise unused
+    /* 0x0198 */ s32 timer;
+    /* 0x019C */ s16 unk_19C; // read, but not written.
```

Comment:
fwiw the actor struct is zero'd when the actor is spawned https://github.com/zeldaret/oot/blob/d8175501adac32dd10918f67f1a7ea278e2ed862/src/code/z_actor.c#L2777

so this is 0 from that, just not explicit
(many actors rely on the struct being zero'd afaik)